pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -920,7 +920,7 @@ func (m *kubeGenericRuntimeManager) doBackOff(pod *v1.Pod, container *v1.Contain
 	key := getStableKey(pod, container)
 	if backOff.IsInBackOffSince(key, ts) {
 		if containerRef, err := kubecontainer.GenerateContainerRef(pod, container); err == nil {
-			m.recorder.Eventf(containerRef, v1.EventTypeWarning, events.BackOffStartContainer, "Back-off restarting failed container")
+			m.recorder.Eventf(containerRef, v1.EventTypeWarning, events.BackOffStartContainer, fmt.Sprintf("Back-off restarting failed container %s", container.Name))
 		}
 		err := fmt.Errorf("back-off %s restarting failed container=%s pod=%s", backOff.Get(key), container.Name, format.Pod(pod))
```

Comment:
Do we want to add the `format.Pod(pod)` as well in line 923?

Reply:
Updated, PTAL thanks.